.github/workflows/aks-auto-start.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
           tenant-id: 531ff96d-0ae9-462a-8d2d-bec7c0b42082 # HMCTS.NET
           allow-no-subscriptions: true
       - name: AKS Auto Start
-        run: ./scripts/aks/auto-start-stop.sh start 
+        run: ./scripts/aks/auto-start-stop.sh
```

Comment:
Will this work with no other changes?
https://github.com/hmcts/auto-shutdown/blob/master/scripts/aks/auto-start-stop.sh#L11C2-L14